openapi/kas-fleet-manager.yaml

```diff
@@ -463,6 +463,50 @@ paths:
         - $ref: '#/components/parameters/page'
         - $ref: '#/components/parameters/size'
         - $ref: '#/components/parameters/instance_type'
+  /api/kafkas_mgmt/v1/instance_types/{cloud_provider}/{cloud_region}:
+    get:
+      summary: Returns the list of supported Kafka instance types and sizes filtered by cloud provider and region
+      operationId: getInstanceTypesByCloudProviderAndRegion
+      security:
+        - Bearer: [ ]
+      responses:
```

Comment:
Missing status 400 here, this endpoint could also return provider/region not supported

Reply:
Thanks @JameelB I'll add 400 status